js/src/redux/providers/status.js

```diff
@@ -48,6 +48,15 @@ export default class Status {
         }
 
         this._store.dispatch(statusBlockNumber(blockNumber));
+
+        this._api.eth
+          .getBlockByNumber(blockNumber)
+          .then((block) => {
+            this._store.dispatch(statusCollection({ gasLimit: block.gasLimit }));
```

Comment:
I was thinking the same. However I would rather not store for the sake of storing - so rather when we have a need for it, come and extract it as opposed to extract it and look for a use at some point down the line.